config/jobs/kubernetes/sig-cloud-provider/azure/release-master.yaml

```diff
@@ -18,7 +18,7 @@ presubmits:
     extra_refs:
       - org: kubernetes-sigs
         repo: cluster-api-provider-azure
-        base_ref: release-1.7
+        base_ref: release-1.8
```

Comment:
A recent test run after this change failed:

https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/directory/pull-kubernetes-e2e-capz-azure-disk/1634192105758789632
@andyzhangx do you think the above test failure has anything to do w/ CAPZ ci-entrypoint.sh?

cc @CecileRobertMichon

Reply:
yes that failure is definitely related. Looks like it's missing the cloud-provider-azure reference

Reply:
taking a look

Reply:
it's using
> Using cluster template: test/dev/cluster-template-custom-builds.yaml

which means it should still be using in-tree, it shouldn't be trying to install the helm chart

Reply:
Revert PR is here: https://github.com/kubernetes/test-infra/pull/28996